dev/arktype.io/docs/api/instanceof.md

```diff
@@ -4,8 +4,19 @@ hide_table_of_contents: true
 
 # instanceOf
 
-## text
+## operator
 
-```ts
-instanceOf: Ark["instanceOf"]
-```
+-   [instanceOf](./instanceof.md)
+
+## helper
+
+-   instanceOf(&lt;object&gt;)
+
+## tuple
+
+-   ["instanceOf", &lt;object&gt;]
+
+## example
+
+-   const instanceOf = type(["instanceOf", Date]) <br/>
+-   const instanceOf = instanceOf(Date) <br/>
```

Comment:
Make sure any of these other variable names that are the same as the name of the helper function are changed